### PR TITLE
Activation 02032023

### PR DESCRIPTION
**Pull request by jasminludolf**

### CODE REVIEW

**Comment by TuurDutoit** [comment]
Hey @jasminludolf! This course has a `mobile` branch - could you create a second PR with the same changes for this branch, as per [our docs](https://datacamp.atlassian.net/wiki/spaces/PRODENG/pages/2428108801/Mobile+content+versioning#Updating-a-course-that-has-a-mobile-version)? Thanks!

**Comment by jasminludolf** [comment]
Hi @TuurDutoit! Yup, no problem. Mobile PR to follow. cc @jsonmak

File: chapter1.md [hunk 1]
```diff
@@ -622,15 +630,9 @@ skills:
...
-
-```
-print("I started with $" + savings + " and now have $" + result + ". Awesome!")
-```
+Suppose, for example, that you've calculated the return of your investment and want to summarize the results in a string.
 
-This will not work, though, as you cannot simply sum strings and integers/floats.
-
-To fix the error, you'll need to explicitly convert the types of your variables. More specifically, you'll need [`str()`](https://docs.python.org/3/library/functions.html#func-str), to convert a value into a string. `str(savings)`, for example, will convert the integer `savings` to a string.
+To do this, you'll need to explicitly convert the types of your variables so that they are all strings. More specifically, you'll need [`str()`](https://docs.python.org/3/library/functions.html#func-str), to convert a value into a string. `str(savings)`, for example, will convert the integer `savings` to a string.
```
Comment (amy-datacamp) [inline chapter1.md:635]: @jasminludolf Please remove the link around the code block.

File: chapter1.md [hunk 2]
```diff
@@ -622,15 +630,9 @@ skills:
...
 
 Similar functions such as [`int()`](https://docs.python.org/3/library/functions.html#int), [`float()`](https://docs.python.org/3/library/functions.html#float) and [`bool()`](https://docs.python.org/3/library/functions.html#bool) will help you convert Python values into any type.
```
Comment (amy-datacamp) [inline chapter1.md:637]: @jasminludolf These links should be changed so they are not associated with code blocks.

File: chapter1.md [hunk 3]
```diff
@@ -640,6 +642,7 @@ Similar functions such as [`int()`](https://docs.python.org/3/library/functions.
 - Convert the variable `pi_string` to a float and store this float as a new variable, `pi_float`.
 
 `@hint`
+- The error message when you _Run Code_ for the first time is telling you that you have an error in line 6 where at least one of your variables is not a string.
 - You should use [`str()`](https://docs.python.org/3/library/functions.html#func-str) twice!
```
Comment (amy-datacamp) [inline chapter1.md:646]: @jasminludolf Similar to the previous comments, the links should be changed so they are not associated with code blocks.

**Pull request merged**